OutOfSchool/OutOfSchool.AuthCommon/Services/ProviderAdminService.cs

```diff
@@ -612,6 +709,32 @@ await providerAdminChangesLogService.SaveChangesLogAsync(providerAdmin, userId,
         return result;
     }
 
+    private List<(string Name, string Value)> GetTrackedUserProperties(User user)
```

Comment:
Why return type `List<(string Name, string Value)>` is used instead of `Dictionary<string, string>`? 
As `Name` is a `User` property name, it is unique and you'll never get key collisions. 
Dictionary is easier for data retrieving (instead of `.FirstOrDefault(x => x.Name == propertyName).Value` simple `TryGetValue` could be used).

Reply:
I changed the List to the Dictionary. Thank you!